### PR TITLE
Update template version

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo-devpack-dotnet/issues/997

using the same as `CompilationEngine`

### CODE REVIEW

**Comment by Jim8y** [comment]
need update

File: src/Neo.SmartContract.Template/templates/neocontractowner/.template.config/template.json [hunk 1]
```diff
@@ -17,10 +17,10 @@
       "datatype": "choice",
       "choices": [
         {
-          "choice": "3.6.2-CI00519"
+          "choice": "3.7.6-*"
         }
       ],
-      "defaultValue": "3.6.2-CI00519",
+      "defaultValue": "3.7.6-*",
```
Comment (shargon) [inline src/Neo.SmartContract.Template/templates/neocontractowner/.template.config/template.json:23]:
```suggestion
      "defaultValue": "3.7.4-*",
```

File: src/Neo.SmartContract.Template/templates/neocontractowner/.template.config/template.json [hunk 2]
```diff
@@ -17,10 +17,10 @@
       "datatype": "choice",
       "choices": [
         {
-          "choice": "3.6.2-CI00519"
+          "choice": "3.7.6-*"
```
Comment (shargon) [inline src/Neo.SmartContract.Template/templates/neocontractowner/.template.config/template.json:20]:
```suggestion
          "choice": "3.7.4-*"
```

File: src/Neo.SmartContract.Template/templates/neocontractoracle/.template.config/template.json [hunk 3]
```diff
@@ -17,10 +17,10 @@
       "datatype": "choice",
       "choices": [
         {
-          "choice": "3.6.2-CI00519"
+          "choice": "3.7.6-*"
         }
       ],
-      "defaultValue": "3.6.2-CI00519",
+      "defaultValue": "3.7.6-*",
```
Comment (shargon) [inline src/Neo.SmartContract.Template/templates/neocontractoracle/.template.config/template.json:23]:
```suggestion
      "defaultValue": "3.7.4-*",
```

File: src/Neo.SmartContract.Template/templates/neocontractoracle/.template.config/template.json [hunk 4]
```diff
@@ -17,10 +17,10 @@
       "datatype": "choice",
       "choices": [
         {
-          "choice": "3.6.2-CI00519"
+          "choice": "3.7.6-*"
```
Comment (shargon) [inline src/Neo.SmartContract.Template/templates/neocontractoracle/.template.config/template.json:20]:
```suggestion
          "choice": "3.7.4-*"
```

File: src/Neo.SmartContract.Template/templates/neocontractnep17/.template.config/template.json [hunk 5]
```diff
@@ -17,10 +17,10 @@
       "datatype": "choice",
       "choices": [
         {
-          "choice": "3.6.2-CI00519"
+          "choice": "3.7.6-*"
         }
       ],
-      "defaultValue": "3.6.2-CI00519",
+      "defaultValue": "3.7.6-*",
```
Comment (shargon) [inline src/Neo.SmartContract.Template/templates/neocontractnep17/.template.config/template.json:23]:
```suggestion
      "defaultValue": "3.7.4-*",
```

File: src/Neo.SmartContract.Template/templates/neocontractnep17/.template.config/template.json [hunk 6]
```diff
@@ -17,10 +17,10 @@
       "datatype": "choice",
       "choices": [
         {
-          "choice": "3.6.2-CI00519"
+          "choice": "3.7.6-*"
```
Comment (shargon) [inline src/Neo.SmartContract.Template/templates/neocontractnep17/.template.config/template.json:20]:
```suggestion
          "choice": "3.7.4-*"
```

File: src/Neo.Compiler.CSharp/CompilationEngine/CompilationEngine.cs [hunk 7]
```diff
@@ -91,7 +91,7 @@ public List<CompilationContext> CompileSources(params string[] sourceFiles)
         {
             return CompileSources(new CompilationSourceReferences()
             {
-                Packages = [new("Neo.SmartContract.Framework", "3.7.4-*")]
+                Packages = [new("Neo.SmartContract.Framework", "3.7.6-*")]
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/CompilationEngine/CompilationEngine.cs:94]:
```suggestion
                Packages = [new("Neo.SmartContract.Framework", "3.7.4-*")]
```

**Review by cschuchardt88 — APPROVED** [review]

**Pull request merged**